backend/middleware/authenticateUser.js

```
@@ -0,0 +1,24 @@
+const jwt = require('jsonwebtoken');
+
+// Middleware to check authentication
+const authenticateUser = (req, res, next) => {
+    let token = req.cookies.token || req.headers.authorization?.split(" ")[1];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'token' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)